ui/admin/app/templates/scopes/scope/credential-stores/credential-store/worker-filter.hbs

```diff
@@ -26,24 +30,43 @@
   </page.actions>
 
   <page.body>
-    <Rose::Layout::Centered>
-      <Hds::ApplicationState as |A|>
-        <A.Header
-          @title={{t 'resources.credential-store.messages.no-filter.title'}}
-        />
-        <A.Body
-          @text={{t
-            'resources.credential-store.messages.no-filter.description'
-          }}
-        />
-        <A.Footer as |F|>
-          <F.LinkStandalone
-            @icon='plus-circle'
-            @text={{t 'resources.credential-store.actions.add'}}
-            @route='scopes.scope.credential-stores.credential-store.index'
+    {{#if @model.worker_filter}}
+      <Hds::CodeBlock
+        data-test-formatted-worker-filter
```

Comment:
I'm noticing this test selector (and the one on line 50) isn't being used anywhere else. Is this perhaps going to be used in a future PR for tests?

Reply:
I will pull these out bc I am opening another PR for tests